setup.py

```diff
@@ -140,6 +140,7 @@ def get_git_sha() -> str:
         "hana": ["hdbcli==2.4.162", "sqlalchemy_hana==0.4.0"],
         "hive": ["pyhive[hive]>=0.6.1", "tableschema", "thrift>=0.11.0, <1.0.0"],
         "impala": ["impyla>0.16.2, <0.17"],
+        "kusto": ["sqlalchemy-kusto>=1.0.1"],
```

Comment:
Just in case, could we restrict to the current major version?
```suggestion
        "kusto": ["sqlalchemy-kusto>=1.0.1, <2"],
```

Reply:
Yes, sounds good to me. We will add this restriction.